styled-slider-view/factory.gradle

```diff
@@ -0,0 +1,22 @@
+// Instead of renaming the artifact, we copy it and rename the copy.
+android.libraryVariants.all { variant ->
```

Comment:
Do we need separate scripts for every module?

Reply:
I think @boyw165 can answer this better. I think yes, this can move to project level.